robot/client/client_session.go

```diff
@@ -28,6 +28,7 @@ var exemptFromSession = map[string]bool{
 	"/proto.rpc.webrtc.v1.SignalingService/OptionalWebRTCConfig":     true,
 	"/proto.rpc.v1.AuthService/Authenticate":                         true,
 	"/proto.rpc.v1.ExternalAuthService/AuthenticateTo":               true,
+	"/viam.robot.v1.RobotService/GetMachineStatus":                   true,
```

Comment:
since we are now using `MachineStatus` for inter-robot communication alongside `ResourceNames`, we need to exempt it from session safety monitoring like we do for `ResourceNames`